egs/ljspeech/TTS/local/prepare_tokens_ljspeech.py

```diff
@@ -45,7 +44,11 @@ def prepare_tokens_ljspeech():
         # Text normalization
         text = tacotron_cleaner.cleaners.custom_english_cleaners(text)
         # Convert to phonemes
-        cut.tokens = g2p(text)
+        tokens_list = phonemize_espeak(text, "en-us")
```

Comment:
At line 42
```
assert len(cut.supervisions) == 1, len(cut.supervisions)
```

Please use
```python3
assert len(cut.supervisions) == 1, (len(cut.supervisions), cut)
```

It is helpful to print the problematic cut on error.